sdk/appconfiguration/azure-data-appconfiguration-v2/customization/src/main/java/AppConfigCustomizations.java

```diff
@@ -0,0 +1,134 @@
+import com.azure.autorest.customization.ClassCustomization;
+import com.azure.autorest.customization.ConstructorCustomization;
+import com.azure.autorest.customization.Customization;
+import com.azure.autorest.customization.LibraryCustomization;
+import com.azure.autorest.customization.PackageCustomization;
+import com.azure.autorest.customization.JavadocCustomization;
+import com.github.javaparser.ast.body.ClassOrInterfaceDeclaration;
+import com.github.javaparser.ast.body.FieldDeclaration;
+import com.github.javaparser.StaticJavaParser;
+import com.github.javaparser.ast.Modifier;
+import com.github.javaparser.ast.NodeList;
+import com.github.javaparser.ast.body.Parameter;
+import com.github.javaparser.ast.stmt.BlockStmt;
+import org.slf4j.Logger;
+import java.util.Arrays;
+
+/**
+ * This class contains the customization code to customize the AutoRest generated code for OpenAI.
+ */
+public class AppConfigCustomizations extends Customization {
+
+    @Override
+    public void customize(LibraryCustomization customization, Logger logger) {
+        PackageCustomization models = customization.getPackage("com.azure.v2.data.appconfiguration.models");
+        PackageCustomization appConfigPackages = customization.getPackage("com.azure.v2.data.appconfiguration");
+        hideClient(appConfigPackages);
+        hideModels(models);
+        renameServiceVersionClassName(appConfigPackages);
+
+//        customizeKeyValueFilter(models.getClass("ConfigurationSettingsFilter"));
+//        customizeKeyValueFields(models.getClass("SettingFields"));
+//        customizeSnapshot(models.getClass("ConfigurationSnapshot"));
```

Comment:
remove?

Reply:
Will need to enable them when LRO is supported in the code gen.